features/fixtures/statistics.json

```diff
@@ -25,6 +25,7 @@
   "details":{
     "document_noun":"statistic",
     "filter":{
+      "content_purpose_supergroup": "research_and_statistics"
```

Comment:
Go to https://finder-frontend-pr-809.herokuapp.com/statistics to view this finder following the changes.